src/main/java/com/vadmack/mongodbtest/controller/ProjectController.java

```diff
@@ -18,6 +19,7 @@
 
     private final ProjectService service;
 
+    @LogRequests
```

Comment:
If you use your own annotation, you need to not forgot to add it every time when you add a new REST API method. Instead of this you can use already existing @GetMappin, @PostMapping and so one